Automattic/woocommerce-payments
# fix: allow all payment methods view when not connected

Fixes https://github.com/Automattic/woocommerce-payments/issues/2327

#### Changes proposed in this Pull Request









When the onboarding is not completed, the merchant cannot go to the "all payment methods" page, due to the redirect put in place to connect the account.

This fixes by preventing the redirect from happening (and still adding the "woocommerce payments" tab) when the account is not connected.

#### Testing instructions





- Ensure your site is disconnected by setting the "Force the plugin to act as disconnected from WCPay" flag in WCPay Dev
- Navigate to WooCommerce > Settings > Payments
- You can see all the payment methods
- Disable the "Force the plugin to act as disconnected from WCPay" flag
- Navigate to WooCommerce > Settings > Payments
- You can see the WCPay settings
- Saving settings/payment methods is still working
- Without these changes, when the site is disconnected, a redirect to the onboarding page happens

-------------------
